app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -66,6 +66,9 @@ Fae.form.ajax = {
         $wrapper.find('.input.file').fileinputer();
       }
 
+      // Re-bind form validations to catch new AJAX loaded fields
+      Fae.form.validator.bindValidationEvents();
```

Comment:
@jclemans We chatted IRT about adding the ability to optionally scope this method so we don't have to rebind all form event handlers every time nested form AJAX is called. Adding as a review item, let me know if you hit any walls on it.

thanks